src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  &__day {
+    position: relative;
+
+    @include daySize ($calendar-day-size);
+    border: 1px solid #000;
```

Comment:
Great job! The only thing I can comment on - is that you could have put the color of the border in a variable, as you did with the background. That's all)